packages/admin/src/FormButtons.tsx

```diff
@@ -0,0 +1,46 @@
+import { Button, Typography } from "@material-ui/core";
+import * as React from "react";
+import { useFormState } from "react-final-form";
+import { FormattedMessage } from "react-intl";
+
+import { useStyles, useThemeProps } from "./FormButtons.styles";
+import { useStackApi } from "./stack/Api";
+
+export const FormButtons = () => {
```

Comment:
@nsams should we call it `FormButtons` or `FormSaveButton`

Reply:
FormSubmitButtons or FormSaveButtons is not really correct, as it also renders Cancel.

- maybe FormSaveCancelButtons?
- And shouldn't we use FinalForm prefix?

Reply:
i tend to `FinalFormSaveCancelButtons`

Reply:
oh, and do we consider this component as "deprecated", only to be used in old, updated apps? (as new ones should always use the new toolbar)

If so, we could add Legacy or something to the name

Reply:
good idea. will add that